src/asserts.js

```diff
@@ -15,6 +15,11 @@
  */
 
 
+// Triple zero width space.
+// This is added to assert error messages, so that we can later identify
+// them, when the only thing that we have is the message. This is the
+// case in many browsers when the global exception handler is invoked.
+export const ASSERT_SENTINEL = '\u200B\u200B\u200B';
```

Comment:
just for my own knowledge, how is this any more reliable than a super random sentinel+prefix?